guides/common/modules/ref_registration-methods.adoc

```diff
@@ -0,0 +1,27 @@
+[id="registration-methods_{context}"]
+= Registration Methods
+
+There are the following methods for registering hosts:
```

Comment:
Alternatives:

* You can use the following registration methods
* Project offers the following registration methods
* Use one of the following methods to register hosts to Project